src/app/features/users-page/users-page.component.scss

```diff
@@ -9,6 +9,7 @@
     }
 }
 
-.table-col {
-    width: 150px;
+.user-table {
+    width: 1000px;
```

Comment:
what about screens less than 1000px?
you need to always set width to 100% and add `max-width: 1000px`